### DISCUSSION TITLE
Question about renewable equipment

### DISCUSSION THREAD

**Discussion by xiaoheimai95** [CURENT/andes#430] [Q&A] [answered]
Dear Dr. Cui and Jinning,
May I ask whether the renewable energy units in this platform belong to the grid-forming or the grid-following control system? Our group wants to use the grid-forming control unit in research.  Looking forward to getting your answer！
Best wishes!

**Comment by jinningwang** [comment, marked as answer]
Hello @xiaoheimai95,

REGCV1 and REGCV2 are implemented with VSG control. REGCF1, REGCF2, and REGCF3 are implementations of EPRI Grid-forming inverter. For the models in the group RenGen, you can do a google search with the model name plus PowerWorld or ESIG to check their control diagram.

I noticed that you have several Q&A in the discussions. Please ***MARK OUT*** an answer to your question if you find any reply appropriate. In this way, the Q&A will be a valuable input to this community that can be helpful for future users.
We commit efforts to maintain this open-source software and we will be grateful if you can also contribute to it if our work really helps you somewhere.

Regards,
Jinning

**Reply by cuihantao** [reply]
Jinning,

Thanks for your reply and maintaining the Q&A!.

@xiaoheimai95 ,

Would you mind letting us know which research group you are with? We are interested in collaborations if you plan to use the tool for research. You can reach me at h.cui@okstate.edu. 

- Hantao

**Reply by xiaoheimai95** [reply]
Thanks Jinning for answering my question. I have already marked answers.
And Dr. Cui, I will contact you via E-mail. Looking forward to the collaborations.
